analyze.go

```diff
@@ -29,6 +29,7 @@ const (
 	ngxConf2More = 0x00002000 // >=2 args
 
 	// some helpful argument style aliases.
+	ngxConfTake012  = ngxConfNoArgs | ngxConfTake1 | ngxConfTake2
```

Comment:
These mostly match the variables listed in the [Development Guide](https://nginx.org/en/docs/dev/development_guide.html#config_directives) (not values, but variables). We don't export these yet but I think it makes sense to do so soon. I'm not so sure we need to add another "alias" and should just do `ngxConfNoArgs | ngxConfTake1 | ngxConfTake2`  in the directive mapping.